blog/releases/2.12/index.mdx

```diff
@@ -0,0 +1,78 @@
+tags: [Releases, License Management, Performance, AI, CI/CD, User Experience]
+date: 2025-06-06
+---
+
+# R2Devops 2.12 Release
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import ReleaseBottomButton from '@site/src/components/ReleaseBottomButton/ReleaseBottomButton.component';
+
+**We are excited to announce R2Devops 2.12! This release brings comprehensive GitLab CI/CD Catalog integration, AWS Bedrock AI provider support, improved issue readability, and enhanced project selection performance.**
+
+{/* truncate */}
+
+## 📦 Comprehensive GitLab CI/CD Catalog Integration
+
+Discover and manage CI/CD catalog components with enhanced visualization:
+
+- **Organization Overview**: Visualize all catalog components across your organization from the R2Devops catalog page
+- **Dedicated Component Pages**: Access comprehensive information for each component including:
+  - Quick-use section with copy-paste pipeline integration examples
+  - Complete input parameter documentation
+  - Full README documentation
+
+![Screenshot of "Components Catalog" on R2Devops](./components_catalog.png)
+
+## 🤖 AWS Bedrock AI Provider Support
+
+Expand your AI capabilities with additional provider options:
+
+- **Multiple AI Providers**: Configure AWS Bedrock as your AI provider alongside existing OpenAI support
+- **Managed AI Services**: Leverage Amazon's managed AI services for issue remediation and recommendations
+- **Flexible Configuration**: Choose AI provider selection to match your organization's infrastructure preferences
+- **AWS Integration**: Seamless integration with your existing AWS environment
+
+![Screenshot of "Bedrock Support" on R2Devops](./bedrock_support.png)
+
+## 📋 Enhanced Issue Readability
+
+Manage issues more efficiently with improved interface design:
+
+- **Dedicated Sidebar**: All issue actions and additional information moved to a focused sidebar
+- **Cleaner Main View**: Streamlined issue list for better readability
+- **Quick Access**: All remediation actions and details easily accessible without cluttering the main interface
+- **Better Organization**: Logical grouping of issue information and available actions
+
+![Screenshot of "Issue Sidebar" on R2Devops](./issue_sidebar.png)
+
+## 🚀 Enhanced Project Selection for Analysis
+
+Experience faster and more intuitive selection of projects to analyze:
+
+- **Improved Performance**: Cached queries significantly reduce loading times
+- **Sticky Group Headers**: Group names remain visible while scrolling through large project lists
+- **Visual Distinction**: Clear differentiation between projects and groups for better navigation
+- **Search Functionality**: Quickly find specific projects in pipeline composition rules
+- **Enhanced User Experience**: Smoother interaction when managing multiple projects
```

Comment:
Same remark. This format doesn't work:
- There is too many point for what the feature is really doing
- Some points are not relevant for user, only for internal development
- Instead of listing feature, describe as short as possible how feature brings value or solve a problem in the user real life